distribution/LICENSE

```diff
@@ -234,9 +234,9 @@ Apache-2.0 licenses
     com.dameng:DmJdbcDriver18 8.1.2.192 Apache-2.0
     com.ecwid.consul:consul-api 1.4.2 Apache-2.0
     com.fasterxml.jackson.core:jackson-annotations 2.12.7 Apache-2.0
-    com.fasterxml.jackson.core:jackson-annotations 2.13.5 Apache-2.0
-    com.fasterxml.jackson.core:jackson-core 2.13.5 Apache-2.0
-    com.fasterxml.jackson.core:jackson-databind 2.13.5 Apache-2.0
+    com.fasterxml.jackson.core:jackson-annotations 2.18.3 Apache-2.0
```

Comment:
Is there a front-end dependency change here?

Reply:
`com.fasterxml.jackson.core:jackson-annotations` modify at #7214